### PR TITLE
New page: aria-hidden

### PR DESCRIPTION
**Pull request by estelle**
- [X] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/accessibility/aria/attributes/aria-hidden/index.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+| {{SpecName("ARIA","#aria-hidden","ARIA: aria-hidden Attribute")}}  | {{Spec2('ARIA')}} |
+
+## See Also
+
+- [`aria-disabled`](/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-disabled)
+- [`aria-modal`](/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-modal)
+- [`aria-expanded`](/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded)
+- HTML {{htmlattrdef('hidden')}} attribute
+- CSS {{CSSXref('display'}} property
+- CSS {{CSSXref('visibility'}} property
```
Comment (LorenzNickel) [inline files/en-us/web/accessibility/aria/attributes/aria-hidden/index.md:100]:
```suggestion
- CSS {{CSSXref('display')}} property
- CSS {{CSSXref('visibility')}} property
```

**Review by LorenzNickel — CHANGES_REQUESTED** [review]

File: files/en-us/web/accessibility/aria/attributes/aria-hidden/index.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+> **Warning:** Do not use `aria-hidden="true"` on focusable elements.
+
+An element's hidden status is based on whether it is rendered. Rendering is usually controlled by CSS. For example, an element whose `display` property is set to `none` is not rendered. An element is considered hidden if it, or any of its ancestors are not rendered or have their `aria-hidden` attribute value set to true.
+
+Use caution when using `aria-hidden` to hide visibly rendered content from assistive technologies. You should not be hiding visible content unless doing so improves the experience for users of assistive technologies by removing redundant or extraneous content. Only when identical or equivalent meaning and functionality is exposed to assistive technologies can removing visible content from the accessibility API be considered.
+
+> **Note:** Consider all disabilities when hiding visibly rendered content from assistive technologies. Not all users of assistive technology are visually impaired. If visible content doesn't match text content in the accessibility API, the user experience will be negatively impacted for sighted users.
```
Comment (ericwbailey) [inline files/en-us/web/accessibility/aria/attributes/aria-hidden/index.md:33]: 💯

File: files/en-us/web/accessibility/aria/attributes/aria-hidden/index.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+Use caution when using `aria-hidden` to hide visibly rendered content from assistive technologies. You should not be hiding visible content unless doing so improves the experience for users of assistive technologies by removing redundant or extraneous content. Only when identical or equivalent meaning and functionality is exposed to assistive technologies can removing visible content from the accessibility API be considered.
+
+> **Note:** Consider all disabilities when hiding visibly rendered content from assistive technologies. Not all users of assistive technology are visually impaired. If visible content doesn't match text content in the accessibility API, the user experience will be negatively impacted for sighted users. 
+
+On the surface, the `aria-hidden="true"` and the `role="presentation"` and its synonoym `role="none"` seem similar, but the intent behind each is different. 
+
+- `aria-hidden="true"` will remove the entire element from the accessibility API. 
+- `role="presentation"` and `role="none"` will remove the semantic meaning of an element while still exposing it and its content to assistive technology.
```
Comment (ericwbailey) [inline files/en-us/web/accessibility/aria/attributes/aria-hidden/index.md:38]: Nice!

File: files/en-us/web/accessibility/aria/attributes/aria-hidden/index.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+- the HTML {{htmlattrdef('hidden')}} attribute is present
+- the element or the element's ancestor is hidden with [`display: none`](/en-US/docs/Web/CSS/display)
+- The element or the element's ancestor is hidden with [`visibility: hidden`](/en-US/docs/Web/CSS/visibility)
+
+In all three scenarios, the attribute is unnecessary to add because the element has already been removed from the accessibility tree. 
+
+Using `aria-hidden="false"` will not re-expose the element to assistive technology if any of its parents specify `aria-hidden="true"`.
+
+When working with dialogs, use the [`aria-modal`](/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-modal) attribute rather than `aria-hidden` to inform assistive technologies that content outside a dialog is inert.
```
Comment (ericwbailey) [inline files/en-us/web/accessibility/aria/attributes/aria-hidden/index.md:50]: I'm confused by this. Are we trying to communicate that non-dialog content will need to be made non-interactive?

File: files/en-us/web/accessibility/aria/attributes/aria-hidden/index.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+```html
+<button type="button" aria-pressed="false">
+  <span class="fa fa-tweet" aria-hidden="true"></span>
+  <span class="label">
+    Tweet
+  </span>
+</button>
+```
+
+We have a button with a Font Awesome icon. We hide the icon from assistive technologies with `aria-hidden="true"` as exposing the icon to assistive technologies could lead to redundancy or, if the icon doesn't have the same content as the visible text, confusion.
```
Comment (ericwbailey) [inline files/en-us/web/accessibility/aria/attributes/aria-hidden/index.md:65]:
```suggestion
We have a button with [a Font Awesome icon](https://fontawesome.com/). We hide the icon from assistive technologies with `aria-hidden="true"`, as exposing the icon to assistive technologies could lead to redundancy or, if the icon doesn't have the same content as the visible text, confusion. 
```

Linking to Font Awesome for those who may be unfamiliar.

**Review by ericwbailey — CHANGES_REQUESTED** [review]
A couple of questions about the `hidden` attribute and modal behavior.

File: files/en-us/web/accessibility/aria/attributes/aria-hidden/index.md (hunk 4 above)
Comment (estelle) [reply files/en-us/web/accessibility/aria/attributes/aria-hidden/index.md]: Made it more clear: "When working with modal dialog popups, ...."

File: files/en-us/web/accessibility/aria/attributes/aria-hidden/index.md [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+On the surface, the `aria-hidden="true"` and the `role="presentation"` and its synonoym `role="none"` seem similar, but the intent behind each is different. 
+
+- `aria-hidden="true"` will remove the entire element from the accessibility API. 
+- `role="presentation"` and `role="none"` will remove the semantic meaning of an element while still exposing it and its content to assistive technology. 
+
+`aria-hidden="true"` should not be added when:
+
+- the HTML {{htmlattrdef('hidden')}} attribute is present
+- the element or the element's ancestor is hidden with [`display: none`](/en-US/docs/Web/CSS/display)
+- The element or the element's ancestor is hidden with [`visibility: hidden`](/en-US/docs/Web/CSS/visibility)
```
[the remark this answers is not in the document]
Comment (estelle) [reply files/en-us/web/accessibility/aria/attributes/aria-hidden/index.md]: added a sentence earlier "hides content from assistive technology but doesn't visually hide anything"

and sentence after "Visually hiding elements with `display` or `visibility` hides content from the screen and from assistive technologies."

File: files/en-us/web/accessibility/aria/attributes/aria-hidden/index.md [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+
+- The HTML {{htmlattrdef('hidden')}} attribute is present
+- The element or the element's ancestor is hidden with [`display: none`](/en-US/docs/Web/CSS/display)
+- The element or the element's ancestor is hidden with [`visibility: hidden`](/en-US/docs/Web/CSS/visibility)
+
+In all three scenarios, the attribute is unnecessary to add because the element has already been removed from the accessibility tree. Visually hiding elements with `display` or `visibility` hides content from the screen and from assistive technologies.
+
+Using `aria-hidden="false"` will not re-expose the element to assistive technology if any of its parents specify `aria-hidden="true"`.
+
+When working with modal dialog popups, use the [`aria-modal`](/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-modal) attribute rather than `aria-hidden` to inform assistive technologies that content outside a dialog is inert.
```
Comment (ericwbailey) [inline files/en-us/web/accessibility/aria/attributes/aria-hidden/index.md:52]:
```suggestion
```

I'd like to remove this section, as a modal will need `aria-modal` to help communicate the presence of a modal, but also requires a lot more to truly work. I'm thinking the steps to discover and remove the non-modal content from both regular screen reader navigation methods and the virtual cursor, and how documenting them here would be way out of scope. 

I worry that keeping this here might communicate that applying only `aria-modal` will be sufficient.

**Review by ericwbailey — CHANGES_REQUESTED** [review]
One last thing!

**Review by ericwbailey — APPROVED** [review]

**Pull request merged**